src/webhook/handlers/tutorial.js

```diff
@@ -0,0 +1,387 @@
+          type: 'image',
+          url: imageUrl,
+          size: 'full',
+          aspectMode: 'cover',
+          aspectRatio: '1:1',
+          gravity: 'top',
+        },
+        {
+          type: 'box',
+          layout: 'vertical',
+          contents: [
+            {
+              type: 'text',
+              text: text,
+              size: 'lg',
+              weight: 'bold',
+              wrap: true,
+            },
+          ],
+          position: 'relative',
+          offsetBottom: '0px',
+          offsetStart: '0px',
+          offsetEnd: '0px',
+          paddingAll: '20px',
+          paddingTop: '18px',
+          flex: 1,
+          justifyContent: 'center',
+        },
+      ],
+      paddingAll: '0px',
+    },
+  };
+}
+
+/**
+ * @param {string} label
+ * @param {string} sessionId
+ * @param {string} postbackState
+ * @returns {object} quickReply items object
+ */
+function createQuickReplyPostbackItem(label, sessionId, postbackState) {
+  return {
+    type: 'action',
+    action: createPostbackAction(label, label, label, sessionId, postbackState),
+  };
+}
+
+/**
+ * @returns {object} Flex message object
+ */
+export function createGreetingMessage() {
+  const text = t`This is a chatbot that looks up suspicious forwarded messages for you. Here is how to use me:`;
+  const imageUrl =
+    'https://uc4e58e1c257cac3657ad4ca0a69.previews.dropboxusercontent.com/p/thumb/AA_IcLRYeJqCzpdE1ZynRVM_LOWI3vkKHByHswDrgZS9BmYfzPao--HbC5krsws4BroUuhF9jQyuXs12XWmO5SuBgPbygVSHWNQM1wrEM_Sa9M-Q6kGD_t_xaXRbNoF2MbBsJfQBSAwJ9XuJsAs2zt8XLURjLX2Kz54W0v6NUuEeHdfetQFlStTzvicxVc_ROqMAYbzXvaE92j1SGHtxJaPZYxDdr5hZhwttLAkXccCgMy052jxgct0DTnSqzO0-cVk6DXRY755iHN_loO_cVlegYQDKmySTfpFMfZp8S8aSoLKsnk2Gqur4f_PabPrWFoMA289DZZ7wRSzdbN8DRlO9cTHo9FoQY-SdcOrpWBq4u6was6uvbXaHeZbrY4ZWmvm8JbrJqQWwb-txBVL1RUzN/p.png?size=800x600&size_mode=3';
+
+  return {
+    type: 'flex',
+    altText: ellipsis(text, 300) + '\n' + FLEX_MESSAGE_ALT_TEXT,
+    contents: {
+      type: 'carousel',
+      contents: [createImageTextBubble(imageUrl, text)],
+    },
+  };
+}
+
+/**
+ * @param {object} data
+ * @returns {object} Flex message object
+ */
+export function createTutorialMessage(data) {
+  const textStep1 = `1. ${t`When receiving a message from elsewhere`}`;
+  const textStep2 = `2. ${t`Long press and share`}`;
+  const textStep3 = `3. ${t`Select Cofacts to share`}`;
+  const textStep4 = `4. ${t`Cofacts replies with a crowd-sourced fact-check or chatbot replies`}`;
+
+  const imageUrlStep1 =
+    'https://uc21ad9bcd486191781c986dfe14.previews.dropboxusercontent.com/p/thumb/AA9R7LsrgHTMDIkz74TvAzOgT5gEcNeCPHWJ_OpIZOYJSBCqzCPMOliCKMe7IRVY_Us_ahRTRgC_d10NkokagS4yRtGo3cGHQfxtEPxRz0Fvo7pxLbEQW3BoM7reJTH0DTbrdku8uTrV3FYz7Evofe55rdRnKGpMPscd9A6YzFeSNvTV31ZwjRvw51utfQG9cwazNCENDVFLeQGl7NANpre3LTKKykTrOuY5t2-VLwYgq6e7fXQGYCGKNuBXWnEljJpPRcD--uDHjsbVtHQFEKt8ap3-lsjoy8AfpmMW619o-pO90Qj3HuRfV8M-_z-JbQS-8cN8Fe0zK-gxMHhRGY_roWKG85vHBePxd7WJDwa--Q/p.png?size=800x600&size_mode=3';
```

Comment:
We may need a better place for these images 
<img width="1019" alt="image" src="https://user-images.githubusercontent.com/108608/97395795-a8687d00-1920-11eb-8731-d5239ed1b6c3.png">